setup.py

```diff
@@ -155,35 +157,35 @@ def initialize_options(self):
         self.skip_flakes = False
         self.skip_test = False
         self.skip_lint = False
-    
+
     def finalize_options(self):
         pass
 
     def run(self):
         pytest_command = ['pytest-3']
         flakes_command = ['pyflakes3', 'repolib']
         lint_command = ['pylint', 'repolib']
-        
+
         if not self.skip_test:
             subprocess.run(pytest_command)
 
         if not self.skip_flakes:
             subprocess.run(flakes_command)
-        
+
         if not self.skip_lint:
             subprocess.run(lint_command)
-            
+
 setup(
-    name = 'repolib',
-    version = get_version(),
-    author = 'Ian Santopietro',
-    author_email = 'isantop@gmail.com',
-    url = 'https://github.com/isantop/repolib',
-    description = 'Easily manage software sources',
-    download_url = 'https://github.com/isantop/repolib/releases',
-    long_description = long_description,
-    tests_require = ['pytest'],
-    license = 'BSD-2',
+    name='repolib',
+    version=get_version(),
+    author='Ian Santopietro',
+    author_email='isantop@gmail.com',
+    url='https://github.com/isantop/repolib',
+    description='Easily manage software sources',
+    download_url='https://github.com/isantop/repolib/releases',
+    long_description=long_description,
+    tests_require=['pytest'],
+    license='BSD-2',
```

Comment:
I noticed this PR mentions fixing the license but the `LGPLv3` change is actually in the other PR.

Reply:
Updated here